Assets/MixedRealityToolkit.Services/InputSystem/InputSystemGlobalListener.cs

```diff
@@ -43,13 +53,13 @@ protected virtual async void Start()
                 }
 
                 lateInitialize = false;
-                MixedRealityToolkit.InputSystem.Register(gameObject);
+                InputSystem?.Register(gameObject);
```

Comment:
>? [](start = 27, length = 1)

Is this ? necessary? It looks like it should be waiting until the InputSystem is non-null (which is why I assume the previous change didn't have the null check)

Reply:
probably not :)

Reply:
removed :)